support/certs/tls.go

```diff
@@ -30,9 +30,11 @@ const (
 	ValidityOneYear  = 365 * ValidityOneDay
 	ValidityTenYears = 10 * ValidityOneYear
 
-	CAHashAnnotation   = "hypershiftlite.openshift.io/ca-hash"
-	CASignerCertMapKey = "ca.crt"
-	CASignerKeyMapKey  = "ca.key"
+	CAHashAnnotation    = "hypershiftlite.openshift.io/ca-hash"
```

Comment:
what's is hypershiftlite?

Reply:
Hmm not sure. It was brought in with https://github.com/openshift/hypershift/pull/1657.

Reply:
The original intent for that annotation was to cause re-generation of the secret if the CA that original signed it had changed. However, looking at the code, that is not currently used. I think it is valuable though. Looks like we don't currently take into consideration the CA when deciding to sign a new cert or not.

Also, have no idea how the hypershiftlite prefix came to this (This is something I worked on as a demo a while back -> https://github.com/openshift-hive/hypershift-lite) 

I think we should address it separately and not in this PR (decide whether to drop or use but with a proper name)